tests/ha/hawk_gui.pm

```diff
@@ -18,66 +18,17 @@ use lockapi;
 use hacluster;
 use x11test;
 use x11utils;
-use version_utils 'is_desktop_installed';
-
-sub install_geckodriver {
-    my $python_ver       = shift;
-    my $arch             = check_var('ARCH', 'x86_64') ? 'linux64' : 'linux32';
-    my $geckodriver_from = "https://github.com/mozilla/geckodriver/releases/download";
-    # Using latest stable version of geckodriver depending on which python is being used
-    my $geckodriver_ver = ($python_ver eq 'python3') ? "v0.24.0" : "v0.17.0";
-    my $geckodriver_pkg = "geckodriver-$geckodriver_ver-$arch.tar.gz";
-
-    assert_script_run "wget -P /tmp $geckodriver_from/$geckodriver_ver/$geckodriver_pkg";
-    type_string "cd /tmp\n";
-    assert_script_run "tar -zxvf $geckodriver_pkg";
-    assert_script_run "mv -i geckodriver /usr/local/bin/";
-    type_string "cd\n";
-}
-
-sub install_required_python_pkgs {
-    my $inst_cmd = '';
-    my $pydeps   = 'selenium paramiko';
+use version_utils;
 
-    assert_script_run "zypper in -y python3-pip || zypper in -y python-pip || zypper in -y python-setuptools";
-
-    # Determine how to install python packages
-    if (is_package_installed('python3-pip') or is_package_installed('python-pip')) {
-        $inst_cmd = 'pip install';
-    }
-    elsif (is_package_installed('python-setuptools')) {
-        $inst_cmd = 'easy_install';
-        # When using easy_install, need to force an older version of urllib3 as the newer ones are broken
-        # for easy_install. Will also use a fixed version of paramiko for the same reason
-        $pydeps = 'selenium paramiko==2.1.6';
-        assert_script_run "$inst_cmd -U urllib3==1.23";
-    }
-    die "Couldn't find a way to install python packages" unless ($inst_cmd);
+sub install_docker {
+    my $docker_url = "https://download.docker.com/linux/static/stable/x86_64/docker-19.03.2.tgz";
```

Comment:
How can we be sure that this version will always be available? If images are archived then that's good for me, but we need to ensure that the test will not fail because the image has been removed, and that we have to retest all, mainly during BV.

Reply:
Yes. These are archived.

https://download.docker.com/linux/static/stable/x86_64/

Reply:
As we discussed with @alvarocarvajald maybe Docker can be installed with SLE+WE+Containers, no?